web/Views/Landlord/Dashboard.cshtml

```diff
@@ -7,62 +7,90 @@
-        </div>
-        <div class="col-7">
+        <div class="col">
             <div class="row">
-                <div class="col-7">
-                    <h6>Hello</h6>
-                    <h2 class="font-blue">@Model.FirstName</h2>
-                    <h6>Start adding your properties to the portal, to get matches</h6>
-                    <div>
-                        <a class="btn btn-primary mt-3" href='@Url.Action("AddNewProperty_Begin", "Property", null)'>Add Properties</a>
-                    </div>
-                </div>
-            </div>
-            <div class="row mt-4">
-                <div class="col-5">
-                    <div class="row">
-                        <div class="col-3">
-                            @if (Model.CharterApproved)
-                            {
-                                <h1>
-                                    <i class="bi bi-check"></i>
-                                </h1>
-                            }
-                            else
-                            {
-                                <h1>
-                                    <i class="bi bi-x"></i>
-                                </h1>
-                            }
-                        </div>
-                        <div class="col-5">
-                            Charter Approval State
-                        </div>
-                    </div>
+                <div class="col-sm-auto">
+                    @* TODO BNH-99: Replace icon with profile picture *@
+                    <img alt="profile picture" src="https://www.softwire.com/wp-content/uploads/2019/07/Gareth-J-Final-Head-Shot.jpg" width="100" height="100" class="rounded-circle img-thumbnail img-fluid"/>
                 </div>
-                <div class="col-5">
+                <div class="col">
                     <div class="row">
-                        <div class="col-3">
-                            <h1>
-                                <i class="bi bi-house-door"></i>
-                            </h1>
-                        </div>
-                        <div class="col-5">
-                            <h1>
-                                @Model.NumOfProperties
-                            </h1>
-                            <h6>Total Properties</h6>
-                        </div>
+                        <h6 class="font-body">Hello</h6>
+                        <h2 class="font-blue font-h1">@Model.CurrentLandlord.FirstName</h2>
+                        @if (Model.CurrentLandlord.NumOfProperties > 0)
+                        {
+                            <h6 class="font-body">View your properties</h6>
+                            <div>
+                                <a class="btn btn-primary btn-lg mt-3 align-middle" href='@Url.Action("ViewProperties", "Landlord", new { id = Model.CurrentLandlord.LandlordId })'>View Properties</a>
+                            </div>
+                        }
+                        else
```

Comment:
Does this distinction between 'View' if they have properties and 'Add' if they don't also work on the Landlord Profile? I know that's not strictly within this ticket, but it's an identical change so might make sense to do it on this branch.